pkg/logs/logs.go

```diff
@@ -99,6 +100,10 @@ func ValidateAndApply(opts *logsapi.LoggingConfiguration) error {
 	return logsapi.ValidateAndApply(opts, nil)
 }
 
+func ValidateAndApplyAsField(opts *logsapi.LoggingConfiguration, fldPath *field.Path) error {
+	return logsapi.ValidateAndApplyAsField(opts, nil, fldPath)
+}
```

Comment:
Is the original  `ValidateAndApply` wrapper function still used anywhere?

I guess this and the original wrapper may be a legacy of the original json logging PR, but it seems hardly worthwhile, since all it does it remove one of the function parameters from the logsapi function.

Reply:
This is still used (search for `ValidateAndApply(`).
I think it indeed might be becoming a bit unnecessary to have these wrapper functions here.
A possible advantage is that we can quickly see what logging functions we use in cert-manager, without having to search for all logs & logsapi occurrences in code.